java-api/src/main/java/com/sap/cloud/security/token/TokenClaims.java

```diff
@@ -30,7 +30,7 @@ private XSUAA() {
 		public static final String ORIGIN = "origin";
 		public static final String GRANT_TYPE = "grant_type"; // OAuth grant type used for token creation
 		public static final String ZONE_ID = "zid"; // tenant GUID -> SAP_GLOBAL_ZONE_ID
-		public static final String CLIENT_ID = "cid"; // OAuth client identifier
+		public static final String CLIENT_ID = "azp"; // OAuth client identifier
```

Comment:
@liga-oz you can keep that and set it to `@deprecated` with a javadoc hint `@link #azp` field

and add a new one for azp